contrib/touched-files-check/src/main.rs

```diff
@@ -7,7 +7,14 @@ fn check_attestations(atts: Vec<&str>, mut keys: HashSet<&str>) -> Result<(), St
         let builder_key = format!("builder-keys/{builder}.gpg");
         keys.remove(&builder_key as &str);
         let builder_key_file = std::fs::File::open(&builder_key)
-            .map_err(|e| format!("Builder key not found for attestation. Attestation: '{att}', Key: '{builder_key}', Error: '{e}'"))?;
+            .map_err(|e| format!("Builder key not found for attestation. Attestation: '{att}', Key: '{builder_key}', Error: '{e}'.\nHelp: Run 'gpg --export --armor {builder} > {builder_key} && git add {builder_key}'"))?;
+        for file in [&builder_key, att, &format!("{att}.asc")] {
+            let content = std::fs::read_to_string(file).unwrap();
+            if !content.chars().all(|c| c.is_ascii()) {
+                return Err(format!("All files must be in ascii format. Make sure to pass --armor to gpg. File: {file}"));
```

Comment:
For cross-reference doc, the `--armor` is set for the asc sig file in:

https://github.com/bitcoin/bitcoin/blob/dd7d82bec03c890b06e189950115bbb0c607f30f/contrib/guix/guix-attest#L253

And required by `cat` to produce a text file in https://github.com/bitcoin/bitcoin/blame/dd7d82bec03c890b06e189950115bbb0c607f30f/doc/release-process.md#L214